cfme/tests/cli/test_appliance_console.py

```diff
@@ -897,8 +897,36 @@ def test_appliance_ssl():
         casecomponent: Appliance
         caseimportance: medium
         initialEstimate: 1/3h
-    """
-    pass
+        testSteps:
+                1. 'cat /opt/rh/cfme-appliance/COPY/etc/manageiq/postgresql.conf.d/
+                01_miq_overrides.conf' run above command and grep to "ssl = on" string
+                2. find the content "hostssl all         all   all     md5" if version > 5.10 in
+                '/opt/rh/cfme-appliance/TEMPLATE/var/lib/pgsql/data/pg_hba.conf' if version  5.11
+                 and use '/opt/rh/cfme-appliance/TEMPLATE/var/opt/rh/rh-postgresql95/lib/pgsql/
+                 data/pg_hba.con' if version 5.10
+        expectedResults:
+                1. Confirm the contents
+                2. Confirm the contents
```

Comment:
Hey @dgaikwad Can you please check indentation for **testSteps** and **expectedResults**?

Reply:
Done